apply_license.py

```diff
@@ -10,28 +17,30 @@
 Unless required by applicable law or agreed to in writing, software distributed
 under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
 CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.'''
-
-extensions = [ ".h", ".c", ".cpp", ".inl", ".m", ".mm", ".sh", ".py", ".java", ".clj", ".go", ".lua" ]
-
-ext_to_comment = {
-    ".h": "// ",
-    ".c": "// ",
-    ".cpp": "// ",
-    ".inl": "// ",
-    ".m": "// ",
-    ".mm": "// ",
-    ".java": "// ",
-    ".sh": "# ",
-    ".py": "# ",
-    ".clj": ";; ",
-    ".go": "// ",
-    ".lua": "-- ",
+specific language governing permissions and limitations under the License.''') % YEAR
+
+def license(comment):
+    return "\n".join([comment + line for line in LICENSE.split("\n")])
+
+# map extensions to strings with the commented license
+ext_to_license = {
+    ".h":    license("// "),
+    ".c":    license("// "),
+    ".cpp":  license("// "),
+    ".inl":  license("// "),
+    ".m":    license("// "),
+    ".mm":   license("// "),
+    ".java": license("// "),
+    ".sh":   license("# "),
+    ".py":   license("# "),
+    ".clj":  license(";; "),
+    ".lua":  license("-- "),
```

Comment:
Do we need to include .script/.gui_script as well?

Reply:
Added